moveit_core/utils/include/moveit/utils/robot_model_test_utils.h

```diff
@@ -182,10 +182,10 @@ class RobotModelBuilder
 
 private:
   /** \brief Adds different collision geometries to a link. */
-  void addLinkCollision(const std::string& link_name, const urdf::CollisionSharedPtr& coll, geometry_msgs::Pose origin);
+  void addLinkCollision(const std::string& link_name, urdf::CollisionSharedPtr coll, geometry_msgs::msg::Pose origin);
```

Comment:
Why were these changed from const references to copies if the shared pointer? The code hasn't changed at all. Why should these?

Reply:
my bad. Fixed